### PR TITLE
More smartstack.yaml docs: explanation of the top level, and moving a…

### PR DESCRIPTION
**Pull request by EvanKrall**
… service to a different location type

### CODE REVIEW

File: docs/source/yelpsoa_configs.rst [hunk 1]
```diff
@@ -279,6 +279,14 @@ Here is an example smartstack.yaml::
       proxy_port: 20028
       timeout_server_ms: 5000
 
+The main key is the service namespace.  Namespaces were introduced for PaaSTA
```
Comment (solarkennedy) [inline docs/source/yelpsoa_configs.rst]: Can you do ``main`` if the intention here reference that string in the config file? At first I thought you meant "main" as in primary

File: docs/source/yelpsoa_configs.rst [hunk 2]
```diff
@@ -279,6 +279,14 @@ Here is an example smartstack.yaml::
...
+services in order to support running multiple daemons from a single service
+codebase. In PaaSTA, each instance in your marathon.yaml maps to a smartstack
+namespace of the same name, unless you specify a different ``nerve_ns``.
```
Comment (solarkennedy) [inline docs/source/yelpsoa_configs.rst]: Can you add the canary example?

**Comment by solarkennedy** [comment]
ship

File: docs/source/yelpsoa_configs.rst (hunk 2 above)
Comment (EvanKrall) [inline docs/source/yelpsoa_configs.rst]: But... a canary generally shares the main namespace.

I could do another example?

File: docs/source/yelpsoa_configs.rst (hunk 1 above)
Comment (EvanKrall) [inline docs/source/yelpsoa_configs.rst]: yeah

File: docs/source/yelpsoa_configs.rst (hunk 2 above)
Comment (solarkennedy) [inline docs/source/yelpsoa_configs.rst]: But the canary *is* and example of your last sentence, where normally the canary would show up on the `canary` namespace, but specifying ``nerve_ns: main`` you override it.

That is just a nice example and documents a common question that people had ("how do you guys do canaries")

**Pull request merged**